rehline/_class.py

```diff
@@ -219,12 +219,13 @@ def make_ReLHLoss(self, X, y, loss={}):
             self.S[1] =   np.sqrt(self.C)
             self.T[0] = np.sqrt(self.C)*y
             self.T[1] = -np.sqrt(self.C)*y
-        elif (self.loss['name'] == 'SVR'):
+        elif (self.loss['name'] in ['SVR', 'svr']):
             self.U = np.ones((2, n))*self.C
-            self.V = np.ones((2, n))*self.loss['eps']
+            self.V = np.ones((2, n))
             self.U[1] = -self.U[1]
-            self.V[0] = self.V[0] - y
-            self.V[1] = self.V[1] + y
+
+            self.V[0] = -self.C*(y + loss['epsilon'])
+            self.V[1] = self.C*(y - loss['epsilon'])
```

Comment:
Thank you for the inputs.

Additionally, a minor correction is required: `loss` should be replaced with `self.loss` in this and the subsequent lines.

Could you please attach the testing results of `tests/_test_svr.py`?

Thank you!

Reply:
Thank you for your great patience and tolerance! I apologize for my clumsiness, but I will try to make more contributions.